Resources/Prototypes/GameRules/events.yml

```diff
@@ -462,7 +462,7 @@
   components:
   - type: StationEvent
     weight: 10
-    earliestStart: 20
+    earliestStart: 5
```

Comment:
```suggestion
```

You silly sausage.

Reply:
ah, right, default already set to 5. silly indeed.